CHANGELOG.md

```diff
@@ -5,11 +5,12 @@ FEATURES:
 * **New Resource:** `vcd_lb_service_monitor`  Load Balancer Service Monitor - [GH-256]
 * **New Data Source:** `vcd_lb_service_monitor` Load Balancer Service Monitor  - [GH-256]
 * **New Resource:** `vcd_edgegateway` creates and deletes edge gateways - [GH-262]
-* **New build command** `make test-env-build` can configure an empty vCD to run the test suite. See `TESTING.md` for details.
+* **New build commands** `make test-env-init` and `make test-env-apply` can configure an empty vCD to run the test suite. See `TESTING.md` for details.
```

Comment:
Not sure about positioning this one (could probably go down in the list due to orderding) and could use (if we have) a [GH-XXX] reference.

Reply:
moved